packages/eslint-config/package.json

```diff
@@ -16,7 +16,7 @@
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-tailwindcss": "^3.13.0",
-    "typescript": "^4.6.4"
+    "typescript": "^4.9"
```

Comment:
Rename it to minor 4.9.5 to be explicit? I think we can also just omit `^` as we talked before. Same below in `peerDependecies` and other package.json files.

Reply:
Ah good idea will do both :)